### PR TITLE
Change to ES6 classes

### PR DESCRIPTION
**Pull request by afaur**
- Remove `js` folder in `src`.
- Use ES6 classes with `React.Component`.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -85,30 +118,47 @@ const Toolbar = React.createClass({
...
+
+class Page extends React.Component {
+
+  constructor(props) {
+
+    super(props)
+
+    this.state = {
+      title: props.title,
+      content: props.content
```
Comment (bayleedev) [inline src/index.js]: Why convert these to state?

File: src/index.js [hunk 2]
```diff
@@ -1,45 +1,60 @@
...
+  }
+
+  static propTypes() {
+    pages: React.PropTypes.array.isRequired
+  }
+
+  getCurrentPageFromUrl() {
     const data = window.location.hash.slice(1).match(/\/page\/(.*)\//)
-    const pageTitle = data[1]
+    const pageTitle = data ? data[1] : 'rawr'
```
Comment (bayleedev) [inline src/index.js]: rawr

File: src/index.js [hunk 3]
```diff
@@ -1,45 +1,60 @@
...
-  componentWillUnmount: function () {
+  }
+
+  componentWillUnmount() {
     window.removeEventListener('hashchange', this.handleHashChange)
-  },
-  handleHashChange: function () {
+  }
+
+  handleHashChange() {
```
Comment (bayleedev) [inline src/index.js]:
``` diff
-  handleHashChange() {
+  handleHashChange = () => {
```

Would fix the bind problem.

File: src/page.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+
+class Page extends Component {
+
+  constructor(props) {
+
+    super(props)
+
```
Comment (bayleedev) [inline src/page.js]: this

has

a lot of

empty lines

Comment (afaur) [inline src/page.js]: ha

ha

ha

okay, I will fix it 👍

**Comment by bayleedev** [comment]
:+1:

**Pull request merged**